docs/examples/driver_examples/QCoDeS example with Galil DMC4133 Controller.ipynb

```diff
@@ -0,0 +1,895 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "bbba7248",
+   "metadata": {},
+   "source": [
+    "# QCoDeS example with Galil DMC4133 Controller"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86f9a772",
+   "metadata": {},
+   "source": [
+    "Purpose of this notebook is to demonstrate how Galil DMC4133 Controller driver along with Arm class can be used for running measurements while controlling the arm head simulteneously. Before begining,\n",
+    "\n",
+    "1. Make sure that you have `gclib` package installed in your environment. If not, then follow the instructions [here](https://www.galil.com/sw/pub/all/doc/gclib/html/python.html) for installation.\n",
+    "2. Make sure that the controller is connected to your PC through an Ethernet cable and the configuration is set as according to these instructions on Windows operating system.\n",
+    "    \n",
+    "        a. Go to Control Panel -> Network and Internet -> Network Connections and select the appropriate network adapter.\n",
+    "        b. Next go to the Properties of that adapter, and then Properties for Internet Protocol Version 4 (TCP/IPv4).\n",
+    "        c. Select \"Use the following IP address\" and add an IP address and Subnet. (If the Galil has an IP address of 10.10.10.100 burned in, you would need a PC IP address of something like 10.10.10.1 with a subnet of 255.255.255.0.)\n",
+    "\n",
+    "Once, the connection to the DMC4133 Controller is established, we can begin with necessary imports and the calibration process. So, let us begin!"
+   ]
+  },
```

Comment:
and would still be nice to have a few words about the vector mode :) at least that "this is submodule that implements the relevant commands, refer to the manual for more info"

Reply:
ok